### DISCUSSION TITLE
Include two checkboxes, for mods Hidden and Classic

### DISCUSSION THREAD

**Discussion by thacer13** [ppy/osu#16011] [Ideas]
1. “Legacy hidden animation” - to make hidden behave like it did up to early 2015 (iirc). In short, the hitcircles would fade normally but only the default-x numbers would fade on top of sliderstarts, which would stay in place throughout the slider.

2. “Classic ScoreV2 mechanics” - SV2 behavior for sliders, wouldn’t be compatible with all of the other Classic options.

**Comment by bdach** [comment]
> SV2 behavior for sliders

can you elaborate what this means? if this means slider head accuracy, then that is already toggleable in the classic mod.

**Reply by thacer13** [reply]
That option you mention makes all the slider ticks work in the new way, I'm looking for slider head accuracy without the new judgement.

**Comment by PercyDan54** [comment]
Is this thread still alive? I'm trying to implement the second but not sure how

**Reply by peppy** [reply]
No idea. Maybe start by explaining what it does, in a better way than attempted above (https://github.com/ppy/osu/discussions/16011#discussioncomment-1774995)

**Reply by smoogipoo** [reply]
We don't reply to every discussion, only the ones which the team actually feels necessary to respond to. This one is not that. If you're interested in implementing this then you're kind of left to your own devices. We may pick it up ourselves/discuss it in the future otherwise if there's large interest (there isn't, yet), and that is why the thread hasn't been answered/closed.

There's plenty of implementation around the classic mod which should give direction for this. We'd need more info to help you.

**Reply by PercyDan54** [reply]
Basically to match stable ScoreV2 behavior, 
`slider.OnlyJudgeNestedObjects = false, slider.head.JudgeAsNormalHitCircle = true`
But NoSliderHeadAccuracy makes both of them true.
So we need a special flag in Classic to get the ScoreV2 behavior
